src/components/action-popover/action-popover-menu/action-popover-menu.component.tsx

```diff
@@ -179,7 +149,7 @@ const ActionPopoverMenu = React.forwardRef<
           let indexValue = focusIndex - 1;
           while (
             indexValue >= firstFocusableItem &&
-            isItemDisabled(indexValue)
+            checkItemDisabled(indexValue)
```

Comment:
suggestion: if you make the change suggested here you can just call `isItemDisabled(items[indexValue)` or if you prefer to keep the memoisation above you could do:

```
  const checkItemDisabled = useCallback(
      (value) => isItemDisabled(items[value]),
      [items]
    );